src/rapids_pre_commit_hooks/shell/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import bashlex
+import bashlex  # type: ignore
```

Comment:
Glad you figured it out!

I think it'd be slightly better to put this into `mypy` configuration instead, so that this doesn't have to be repeated for each import of `bashlex`.

Using the blunt `ignore_missing_imports` does this globally ("ignore any imports you can't find type hints for")

```toml
[tool.mypy]
ignore_missing_imports = true
```

Or for just bashlex

```toml
[tool.mypy-bashlex.*]
ignore_missing_imports = true
```

This is described in https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker.